soda_machine.rb

```diff
@@ -7,12 +7,20 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+   return @sodas.length
   end
 
   def find_soda(soda_brand)
+    if @sodas.brand == soda_brand
+      return @sodas.soda_brand
```

Comment:
What is `@sodas`? As in, what would you get if you called `.class` on it?  Does it have an attribute called `soda_brand`?